### PR TITLE
Update app-discovered-apps.md

### PR DESCRIPTION
**Pull request by msbemba**
reduced ambiguity by correcting descriptions for For company-owned devices	(Android enterprise)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2445

**Comment by PRMerger6** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/apps/app-discovered-apps.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The following list provides the app platform type, the apps that are monitored f
 | iOS/iPadOS | Only managed apps | All apps installed on the device | Every 7 days from device enrollment |
 | macOS | Only managed apps | All apps installed on the device | Every 7 days from device enrollment |
 | Android | Only managed apps | All apps installed on the device | Every 7 days from device enrollment |
-| Android Enterprise | Only managed apps | Only apps installed in the Android Enterprise fully managed work profile | Every 7 days from device enrollment |
+| Android Enterprise | Only managed apps | Only apps installed in the Android Enterprise work profile device | Every 7 days from device enrollment |
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-discovered-apps.md:68]:
```suggestion
| Android Enterprise | Only managed apps | Only apps installed on the Android Enterprise work profile device | Every 7 days from device enrollment |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@Erikre Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Review by Erikre — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**